internal/net/control.go

```diff
@@ -37,7 +37,7 @@ func NewGRPCListener(l log.Logger, s Service, controlAddr string) (ControlListen
 	}
 
 	proto.RegisterControlServer(grpcServer, s)
-	pdkg.RegisterDKGControlServer(grpcServer, s)
+	pdkg.RegisterDKGPublicServer(grpcServer, s)
```

Comment:
I would have expected a control server here, no?

Reply:
ah wait - now I'm slightly confused, I thought this was for the public listener but indeed `controlAddr` is above hmm